core/payment-driver/base/src/bus.rs

```diff
@@ -111,6 +111,7 @@ pub async fn list_unlocked_identities() -> Result<Vec<NodeId>, GenericError> {
     Ok(unlocked_list)
 }
 
+/// Notifies the Payment service that the driver is ready.
```

Comment:
Nope. It notifies the payment service that an **account** is ready (for sending/receiving funds - see mode).